internal/cmd/create.go

```diff
@@ -0,0 +1,244 @@
+		Example:           fmt.Sprintf("%s create \"pkg:apk/wolfi/git@2.39.0-r1?arch=x86_64\" CVE-2022-39260 fixed ", appname),
+		SilenceUsage:      false,
+		SilenceErrors:     false,
+		PersistentPreRunE: initLogging,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := opts.Validate(args); err != nil {
+				return err
+			}
+			// If we have arguments, add them
+			for i := range args {
+				switch i {
+				case 0:
+					opts.Products = append(opts.Products, args[i])
+				case 1:
+					opts.Vulnerability = args[i]
+				case 2:
+					opts.Status = args[i]
+				}
+			}
+			newDoc := vex.New()
+
+			statement := vex.Statement{
+				Vulnerability:   opts.Vulnerability,
+				Products:        opts.Products,
+				Subcomponents:   opts.Subcomponents,
+				Status:          vex.Status(opts.Status),
+				StatusNotes:     opts.StatusNotes,
+				Justification:   vex.Justification(opts.Justification),
+				ImpactStatement: opts.ImpactStatement,
+			}
+			newDoc.Statements = append(newDoc.Statements, statement)
+			if _, err := newDoc.GenerateCanonicalID(); err != nil {
+				return fmt.Errorf("generating document id: %w", err)
+			}
+
+			out := os.Stdout
+
+			if opts.outFilePath != "" {
+				f, err := os.Create(opts.outFilePath)
+				if err != nil {
+					return fmt.Errorf("opening vex file to write document: %w", err)
+				}
+				out = f
+				defer f.Close()
+			}
+
+			if err := newDoc.ToJSON(out); err != nil {
+				return fmt.Errorf("writing new vex document: %w", err)
+			}
+
+			if opts.outFilePath != "" {
+				fmt.Fprintf(os.Stderr, " > VEX document written to %s\n", opts.outFilePath)
+			}
+			return nil
+		},
+	}
+
+	createCmd.PersistentFlags().StringVar(
+		&opts.DocumentID,
+		"id",
+		"",
+		"ID for the new VEX document (default will be computed)",
+	)
+
+	createCmd.PersistentFlags().StringVar(
+		&opts.Author,
+		"author",
+		vex.DefaultAuthor,
+		"author to record in the new document",
+	)
+
+	createCmd.PersistentFlags().StringVar(
+		&opts.AuthorRole,
+		"author-role",
+		vex.DefaultRole,
+		"author role to record in the new document",
+	)
+
+	createCmd.PersistentFlags().StringVarP(
+		&opts.Vulnerability,
+		"vuln",
+		"v",
+		"",
+		"vulnerability to add to the statement (eg CVE-2023-12345)",
+	)
+
+	createCmd.PersistentFlags().StringSliceVarP(
+		&opts.Products,
+		"product",
+		"p",
+		[]string{},
+		"list of products to list in the statement, at least one is required",
+	)
+
+	createCmd.PersistentFlags().StringVarP(
+		&opts.Status,
+		"status",
+		"s",
+		"",
+		fmt.Sprintf("status of the product vs the vulnerability, see %s show statuses for list", appname),
+	)
+
+	createCmd.PersistentFlags().StringSliceVar(
+		&opts.Products,
+		"subcomponents",
+		[]string{},
+		"list of subcomponents to add to the statement",
+	)
+
+	createCmd.PersistentFlags().StringVarP(
+		&opts.Justification,
+		"justification",
+		"j",
+		"",
+		fmt.Sprintf("justification for not_affected status, see %s show justifications for list", appname),
```

Comment:
```suggestion
		fmt.Sprintf("justification for not_affected status, see `%s show justifications` for list", appname),
```

Maybe not this suggestion exactly, but somehow to make it clear which words are part of the command we're suggesting they run...

Reply:
Sounds good enough to me, i've added the backticks